File: src/com/dafttech/terra/world/tiles/TileStone.java
package com.dafttech.terra.world.tiles;

import com.badlogic.gdx.graphics.g2d.TextureRegion;
import com.dafttech.terra.resources.Resources;

public class TileStone extends Tile {

    public TileStone() {
        super();
    }

    @Override
    public TextureRegion getImage() {
        return Resources.TILES.getImage("stone");
    }

}

File: terra/src/main/scala/com/dafttech/terra/game/world/tiles/TileStone.scala
package com.dafttech.terra.game.world.tiles

import com.badlogic.gdx.graphics.g2d.TextureRegion
import com.dafttech.terra.resources.Resources
import monix.eval.Task

class TileStone() extends Tile {
  override val getImage: Task[TextureRegion] = Resources.TILES.getImage("stone")
}